lms/djangoapps/instructor_task/tests/test_tasks_helper.py

```diff
@@ -869,60 +869,6 @@ def test_cohort_content(self):
         expected_grades = [self._format_user_grade(header_row, **user_grade) for user_grade in user_grades]
         self.verify_rows_in_csv(expected_grades)
 
-    @patch('courseware.grades.MaxScoresCache.get', Mock(return_value=1))
-    def test_cohort_content_with_maxcache(self):
-        """
-        Tests the cohoted course grading to test the scenario in which `max_scores_cache` is set for the course
-        problems.
-        """
```

Comment:
ok.  Also, I notice that the unit test right above, `test_cohort_content`, tests the same functionality without the max_cache override.  So that coverage should continue to exist.